File: tck/scala-tck/src/main/scala/kalix/tck/model/view/ViewTckSourceEntity.scala
/*
 * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
 */

package kalix.tck.model.view

import kalix.scalasdk.valueentity.ValueEntityContext

/** A value entity. */
class ViewTckSourceEntity(context: ValueEntityContext) extends AbstractViewTckSourceEntity {
  override def emptyState: Ignore = Ignore.defaultInstance

}

File: tck/java-tck/src/main/java/kalix/tck/model/view/ViewTckSourceEntity.java
/*
 * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
 */

package kalix.tck.model.view;

import kalix.javasdk.valueentity.ValueEntityContext;

/** A value entity. */
public class ViewTckSourceEntity extends AbstractViewTckSourceEntity {
  @SuppressWarnings("unused")
  private final String entityId;

  public ViewTckSourceEntity(ValueEntityContext context) {
    this.entityId = context.entityId();
  }

  @Override
  public View.Ignore emptyState() {
    return null;
  }
}
